### PR TITLE
GOV-313 Vouchers deployment file

### PR DESCRIPTION
**Pull request by apurbraj**
## Description

* Adding deployment files for voucher [GOV-313](https://mifosforge.jira.com/browse/GOV-313?atlOrigin=eyJpIjoiODBlMzMzYmJiYzFmNDY2ZmFlZmRmNTE5MTJhNjBlYzIiLCJwIjoiaiJ9)
* Add a link to teh design document or include the design bullet points related to this PR here.

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

[GOV-313]: https://mifosforge.jira.com/browse/GOV-313?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: helm/ph-ee-engine/vouchers/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# connector_bulk
```
Comment (fynmanoj) [inline helm/ph-ee-engine/vouchers/README.md:1]: 👍  documentation

File: helm/ph-ee-engine/vouchers/templates/clusterrole.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{- if .Values.managedServiceAccount }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  # "namespace" omitted since ClusterRoles are not namespaced
+  name: pph-ee-vouchers-c-role
```
Comment (fynmanoj) [inline helm/ph-ee-engine/vouchers/templates/clusterrole.yaml:6]: we have changed this way of creating the role, please refer Ankita's PR for GOV 272

File: helm/ph-ee-engine/vouchers/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            - name: ZEEBE_CLIENT_MAX_EXECUTION_THREADS
+              value: "{{ .Values.zeebe.client.max-execution-threads }}"
+            - name: ZEEBE_CLIENT_EVENLY_ALLOCATED_MAX_JOBS
+              value: "{{ .Values.zeebe.client.evenly-allocated-max-jobs }}"
+            - name: ZEEBE_BROKER_CONTACTPOINT
+              value: "{{ .Values.zeebe.broker.contactpoint }}"
+{{- if .Values.extraEnvs | default .Values.deployment.extraEnvs }}
+{{ toYaml ( .Values.extraEnvs | default .Values.deployment.extraEnvs ) | indent 10 }}
+{{- end }}        
+          envFrom: {{ toYaml ( .Values.envFrom | default .Values.deployment.envFrom ) | nindent 12 }}
```
Comment (fynmanoj) [inline helm/ph-ee-engine/vouchers/templates/deployment.yaml:78]: can you try adding Liveness and readiness check

File: helm/ph-ee-engine/vouchers/templates/role.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{{- if .Values.managedServiceAccount }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: ph-ee-vouchers-role
```
Comment (fynmanoj) [inline helm/ph-ee-engine/vouchers/templates/role.yaml:5]: we have changed this way of creating the role

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Comment by fynmanoj** [comment]
JIRA Id

File: helm/g2p-sandbox/values.yaml [hunk 5]
```diff
@@ -501,6 +503,64 @@ ph-ee-engine:
...
+    deployment:
+      annotations:
+        deployTime: "{{ .Values.deployTime }}"
+    spring:
+      datasource:
+        url: jdbc:mysql://operationsmysql:3306/voucher_management
+        username: mifos
+        password: password
+
+    logging:
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:535]: remove dead and duplicate code

File: helm/g2p-sandbox/values.yaml [hunk 6]
```diff
@@ -501,6 +503,64 @@ ph-ee-engine:
...
+      queue_capacity: 100
+
+    expiry_time: 60
+
+    payer:
+      tenant: rhino
+      identifier: 12345678
+      identifierType: MSISDN
+    voucher:
+      hostname: "https://vouchers.sandbox.fynarfin.io"
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:551]: use mifos.io

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Review by fynmanoj — APPROVED** [review]

File: helm/ph-ee-engine/vouchers/templates/deployment.yaml (hunk 3 above)
Comment (apurbraj) [reply helm/ph-ee-engine/vouchers/templates/deployment.yaml:76]: When adding Liveness and readiness check pod was not coming up

**Pull request merged**